content/docs/faq-ajax.md

```diff
@@ -50,9 +51,9 @@ class MyComponent extends React.Component {
             items: result.items
           });
         },
-        // Note: it's important to handle errors here
-        // instead of a catch() block so that we don't swallow
-        // exceptions from actual bugs in components.
+        // שים לב: חשוב לטפל בשגיאות כאן
+        // כדי שלא נפספס catch()-במקום הבלוק
+        // חריגות מבאגים בפועל בקומפוננטות שלנו.
```

Comment:
As I understand this, this comment basically says that if we handle errors inside of a catch statement, any error that we get from a component will be "swallowed".
I suggest this:
על מנת שלא "נבלע" חריגות מבאגים בקומפוננטות שלנו. 

I omitted 'בפועל' because imo its confusing and has the same meaning if omitted